[CI] CoordinatorTests.testLogsMessagesIfPublicationDelayed fails reproducibly

I had this failing on a PR and it reproduces on master for me:

https://gradle-enterprise.elastic.co/s/s33wgvgtlqvks

```
./gradlew ':server:test' --tests "org.elasticsearch.cluster.coordination.CoordinatorTests.testLogsMessagesIfPublicationDelayed" -Dtests.seed=BACFE347DDD2C688 -Dtests.security.manager=true -Dtests.locale=fr-CH -Dtests.timezone=Asia/Karachi -Druntime.java=11
```

Error:
```
java.lang.AssertionError: expected to see lag warning but did not
    Expected: <true>
         but: was <false>
        at __randomizedtesting.SeedInfo.seed([BACFE347DDD2C688:50A5B1B04E87756]:0)
        at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
        at org.elasticsearch.test.MockLogAppender$SeenEventExpectation.assertMatched(MockLogAppender.java:132)
        at org.elasticsearch.test.MockLogAppender.assertAllExpectationsMatched(MockLogAppender.java:66)
        at org.elasticsearch.cluster.coordination.CoordinatorTests.testLogsMessagesIfPublicationDelayed(CoordinatorTests.java:1513)
```

I saw this starting to fail on Aug 29th in CI failure mails now on 7.x and master, will probably push a commit to mute this test.

--- a/server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java
+++ b/server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java
@@ -1462,7 +1462,6 @@ public class CoordinatorTests extends AbstractCoordinatorTestCase {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/62383")
     public void testLogsMessagesIfPublicationDelayed() throws IllegalAccessException {
         try (Cluster cluster = new Cluster(between(3, 5))) {
             cluster.runRandomly();
@@ -1508,7 +1507,7 @@ public class CoordinatorTests extends AbstractCoordinatorTestCase {
                     });
                 cluster.getAnyLeader().submitValue(randomLong());
                 cluster.runFor(defaultMillis(PUBLISH_TIMEOUT_SETTING) + 2 * DEFAULT_DELAY_VARIABILITY
-                        + defaultMillis(LagDetector.CLUSTER_FOLLOWER_LAG_TIMEOUT_SETTING),
+                        + defaultMillis(LagDetector.CLUSTER_FOLLOWER_LAG_TIMEOUT_SETTING) + DEFAULT_DELAY_VARIABILITY,
                     "waiting for messages to be emitted");
 
                 mockLogAppender.assertAllExpectationsMatched();

